src/Examine.Core/Search/IExamineSpatialPoint.cs

```diff
@@ -0,0 +1,18 @@
+namespace Examine.Search
+{
+    /// <summary>
+    /// Spatial Point Shape
+    /// </summary>
+    public interface IExamineSpatialPoint : IExamineSpatialShape
```

Comment:
Does these need to include `IExamine` in name as they exists under `Examine` nanespace? Although we have `IExamineManager`.

This seems a but simpler though;
`public interface ISpatialPoint : ISpatialShape`

Should it exists under a `Examine.Search.Spatial` namespace instead to separate Spatial search logic?

Reply:
It seems the facets were include under `Examine.Search` namespace and not `Examine.Search.Facets` though.

The Lucene.NET library included facets under `Lucene.Net.Facet`.

Reply:
Done